src/bin/rav1e.rs

```diff
@@ -266,20 +279,26 @@ fn run() -> Result<(), error::CliError> {
   cli.enc.time_base = video_info.time_base;
   let cfg = Config { enc: cli.enc, threads: cli.threads };
 
-  eprintln!(
-    "{}x{} @ {}/{} fps",
+  cli.io.output.write_header(
     video_info.width,
     video_info.height,
-    video_info.time_base.den,
-    video_info.time_base.num
+    video_info.time_base.den as usize,
+    video_info.time_base.num as usize,
   );
 
-  cli.io.output.write_header(
+  info!(
+    "Using y4m decoder: {}x{}p @ {}/{} fps, {}, {}-bit",
     video_info.width,
     video_info.height,
-    video_info.time_base.den as usize,
-    video_info.time_base.num as usize,
+    video_info.time_base.den,
+    video_info.time_base.num,
+    video_info.chroma_sampling,
+    video_info.bit_depth
   );
+  info!("CPU Feature Level: {}", cfg.enc.cpu_feature_level);
+  if cli.verbose {
+    info!("Encoding settings: {}", cfg.enc.print_settings());
```

Comment:
Would it be useful to print this always? It's a long string, but the number of times someone has said "I didn't realize feature X isn't enabled at speed 5" is a lot.

Reply:
Probably it is a good idea.